KthLastElement.java

```diff
@@ -0,0 +1,46 @@
+package com.katermar;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Created by USER on 5/15/2017.
+ */
+public class KthLastElement {

```

Comment:
Small comment from design/naming perspective: it is usually good to try to generalize (within reason, of course) classes and name them appropriately.

In this case there is a class KthLastElement, which is de facto an implementation of a linked list node, and as such could be used for many different things, like finding given value in the list, removing one etc. Unfortunately its name suggests that the only thing it could ever be good for is finding k-th last element.

It would be much nicer to extract linked list functionality (storing value and pointer to next) to separate class, called Node, Element or something similar, and have KthLastElement.findKth operate on it.